charts/hono/templates/NOTES.txt

```diff
@@ -14,41 +14,41 @@ The output should look similar to this:
 
 NAME                                       {{ "READY   STATUS    RESTARTS   AGE" | indent ( len .Release.Name ) }}
 {{- if .Values.adapters.amqp.enabled }}
-{{ .Release.Name }}-adapter-amqp-65cfb4d675-g5wn4             1/1     Running   0          5m50s
+{{ include "hono.fullname" . }}-adapter-amqp-65cfb4d675-g5wn4             1/1     Running   0          5m50s
```

Comment:
Actually, I think we could get rid of this whole rather obscure `{{- $_ := set $ "dot" $args.dot }}` usage, by reverting the `define "hono.fullname"` named template changes - not requiring a `dot` dict there anymore but letting "hono.fullname" use `.Values` directly, instead of `.dot.Values`.

That would mean:
- replacing `include "hono.fullname" $args` with `include "hono.fullname" .` in the yaml files
- reverting the above `$_ := set $ "dot"` changes
- replacing `include "hono.fullname" .` with `include "hono.fullname" .dot` in `_helpers.tpl`
- adapting the place where `$kafkaChartDotScope` is defined within `_helpers.tpl`, removing the  outer `dict "dot"`
- replacing `include "hono.fullname" (dict "dot" .)` with `include "hono.fullname" .` in `values.yaml`
- adapting `kafka-example-keys.yaml`, using this in the `metadata` block:
```
  {{/* metadata.name is set to "{fullNameOfKafkaSubChart}-example-keys" here, meaning e.g. "{ReleaseName}-kafka-example-keys", not "{ReleaseName}-hono-example-keys" */}}
  {{- $kafkaNameValues := pick .Values.kafka "nameOverride" "fullnameOverride" }}
  {{- $kafkaChartDotScope := dict "Release" .Release "Chart" (dict "Name" "kafka") "Values" $kafkaNameValues }}
  {{- $args := dict "dot" ($kafkaChartDotScope) "component" "kafka" "name" "example-keys" }}
  {{- include "hono.metadata" $args | nindent 2 }}
```

@WatcherWhale @sophokles73 WDYT?

Reply:
It is indeed a rather obscure line, I'm in favor of changing it.

Reply:
The files
`charts/hono/templates/example-data-grid/configmap.yaml`
and
`charts/hono/templates/hono-service-device-registry-base/hono-service-device-registry-post-install-configmap.yaml`
still need to be adapted (`{{- $_ := set $ "dot" $args.dot }}` still in there).

Reply:
In these 2 files, can you also restore usage of `.` instead of `$` in the `{{- tpl ( .Files.Get ...` line?